affiliatewp-order-details-for-affiliates.php

```diff
@@ -170,35 +197,40 @@ private function hooks() {
 	}
 
 	/**
-	 * Redirect affiliate to main dashboard page if they cannot access order details tab
+	 * Redirect affiliate to main dashboard page if they cannot access order details tab.
```

Comment:
```suggestion
	 * Redirects affiliate to main dashboard page if they cannot access order details tab.
```
Third-person singular verb in a DocBlock summary.